text.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
Good job @sja-thedude for creating a markdown file. As the content of this file suggests, this file should be added in `.gitignore`.